README.md

```diff
@@ -1,57 +1,62 @@
-
-Similar to cw20-ics20, the contract starts with minimal state, just the default timeout for all the packets it sends.
-
-An external party has to make one or more channels using this contract as one endpoint.
-
-You can send any SG721 token to this contract via the [receiver pattern](https://github.com/CosmWasm/cw-nfts/blob/main/packages/cw721/src/receiver.rs).
-
-## Messages
-
-This contract only accepts a `Sg721ReceiveMsg` from a sg721 contract.
-
-```rust
-pub struct Sg721ReceiveMsg {
-    pub sender: String,
-    pub token_id: String,
-    pub msg: Binary,
-}
-```
-
-The data inside the message must be JSON-serialized.
-
-```rust
-pub struct TransferMsg {
-    /// The local channel to send the packets on
-    pub channel: String,
-    /// The remote address to send to
-    /// Don't use HumanAddress as this will likely have a different Bech32 prefix than we use
-    /// and cannot be validated locally
-    pub remote_address: String,
-    /// How long the packet lives in seconds. If not specified, use default_timeout
-    pub timeout: Option<u64>,
-}
-```
-
-## Queries
-
-Queries only make sense relative to the established channels of this contract.
-
-- `Port{}` - returns the port ID this contract has bound, so you can create channels. This info can be queried
-  via wasmd contract info query, but we expose another query here for convenience.
-- `ListChannels{}` - returns a (currently unpaginated) list of all channels that have been created on this contract.
-  Returns their local channelId along with some basic metadata, like the remote port/channel and the connection they
-  run on top of.
-- `Channel{id}` - returns more detailed information on one specific channel. In addition to the information available
-  in the list view, it returns the list of class_ids (which include the contract address).
-- `Tokens{channel_id, class_id}` - returns all the tokens that have been sent through the channel for a specific class-id.
-
-## Credits
-
-This README was adapted from the cw20-ics20 [README](https://github.com/CosmWasm/cw-plus/tree/main/contracts/cw20-ics20).
+This is an implementation of the [ICS 721
+specification](https://github.com/cosmos/ibc/tree/master/spec/app/ics-721-nft-transfer)
+written in CosmWasm.
+
+**This code has not yet been audited. Please, do not use it for
+anything mission critical.**
+
+This is an extended implementation of [ICS
+721](https://github.com/cosmos/ibc/tree/master/spec/app/ics-721-nft-transfer)
+to make ICS 721 NFTs
+[cw721](https://github.com/CosmWasm/cw-nfts/tree/main/packages/cw721)
+compatible. This means that any dapp that interacts with cw721 NFTs
+can also interact with NFTs from other blockchains when this
+implementation of ICS 721 is on the receiving side of the transfer.
+
+Three contracts orchestrate this:
+
+1. `cw-ics721-bridge` implements the "NFT transfer bridge" and "NFT
+   asset tracking module" parts of the ICS 721 spec.
+2. `ics-escrow` escrows NFTs while are away on foreign chains.
```

Comment:
Looks like we deleted ics-escrow related stuff so we don't need this in the readme